tests/search/test_main.py

```diff
@@ -75,12 +75,14 @@ def test_search_input(
     search_input = page.get_by_placeholder("Search here...")
     expect(search_input).to_be_visible()
     search_input.fill("mex")
+    search_input.press("Enter")
     expect(page.get_by_text("Showing 1 of 1 items")).to_be_visible()
     page.screenshot(
         path="tests_search_test_main-test_search_input-on-search-input-1-found.png"
     )
 
     search_input.fill("totally random search dPhGDHu3uiEcU6VNNs0UA74bBdubC3")
+    search_input.press("Enter")
```

Comment:
could you mix it up, 1 test with Enter and 1 test with clicking on the button?

Reply:
done in cf97b7a664f9b39d774ec9e34feb059d343539c2